Makefile

```diff
@@ -277,13 +280,15 @@ manifests:
 
 # lint/test/etc
 
-$(HOME)/go/bin/golangci-lint:
+$(GOPATH)/bin/golangci-lint:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b `go env GOPATH`/bin v1.23.8
 
 .PHONY: lint
-lint: server/static/files.go $(HOME)/go/bin/golangci-lint
+lint: server/static/files.go $(GOPATH)/bin/golangci-lint
 	# Tidy Go modules
 	go mod tidy
+	# Build to find errors that will prevent linting.
+	go build ./...
```

Comment:
Is there a reason you added this?

I thought that the linter itself detects build errors. Seems like this line would just make `make lint` take longer

Reply:
I'll revert this line- it's not needed.

Reply:
Ops - I don't want to revert - there's a class of builds errors that break the linter - when SSA cannot be created. This prevents this. It does assume we want to build it mind you - not always the case.